lib/src/services/api.dart

```diff
@@ -1,5 +1,6 @@
 import 'dart:async';
 import 'dart:convert';
+import 'package:hot_recharge/src/models/end_user_balance.dart';
```

Comment:
lets try to use `relative path import` here as suggested in the [docs here](https://dart.dev/guides/libraries/create-library-packages#importing-library-files). You might want to add all models in `../models/index.dart` to avoid many imports in one file